core/services/pipeline/runner_test.go

```diff
@@ -57,15 +57,15 @@ func Test_PipelineRunner_ExecuteTaskRuns(t *testing.T) {
 
 	d := pipeline.TaskDAG{}
 	s := fmt.Sprintf(`
-ds1 [type=bridge name="example-bridge" timeout=0 requestData="{\"data\": {\"coin\": \"BTC\", \"market\": \"USD\"}}"]
+ds1 [type=bridge name="example-bridge" timeout=0 requestData=<{"data": {"coin": "BTC", "market": "USD"}}>]
 ds1_parse [type=jsonparse lax=false  path="data,result"]
 ds1_multiply [type=multiply times=1000000000000000000]
 
-ds2 [type=http method="GET" url="%s" requestData="{\"data\": {\"coin\": \"BTC\", \"market\": \"USD\"}}"]
+ds2 [type=http method="GET" url="%s" requestData=<{"data": {"coin": "BTC", "market": "USD"}}>]
```

Comment:
Now let's see a test where one of the json keys has an angle bracket in it ;)

Reply:
Shhh! (adding)

Reply:
It's not considered valid DOT to do that, so anyone who needs `<>` inside of a string should use "these quotes"